pkg/kstatus/watcher/dynamic_informer_factory.go

```diff
@@ -55,3 +57,34 @@ func (f *DynamicInformerFactory) NewInformer(ctx context.Context, mapping *meta.
 		f.Indexers,
 	)
 }
+
```

Comment:
Suggest adding TODO to remove regex once client-go is updated to a version including the fix https://github.com/kubernetes/kubernetes/pull/110076 (1.25+).

Reply:
done